src/cascadia/TerminalSettingsModel/defaults.json

```diff
@@ -343,7 +343,12 @@
         { "command": { "action": "moveFocus", "direction": "left" }, "keys": "alt+left" },
         { "command": { "action": "moveFocus", "direction": "right" }, "keys": "alt+right" },
         { "command": { "action": "moveFocus", "direction": "up" }, "keys": "alt+up" },
-        { "command": { "action": "moveFocus", "direction": "previous" }, "keys": "ctrl+alt+left" },
+        { "command": { "action": "moveFocus", "direction": "previous" }, "keys": "ctrl+alt+left"},
+        { "command": { "action": "movePane", "direction": "down" }, "keys": "alt+j" },
+        { "command": { "action": "movePane", "direction": "left" }, "keys": "alt+h" },
+        { "command": { "action": "movePane", "direction": "right" }, "keys": "alt+l" },
+        { "command": { "action": "movePane", "direction": "up" }, "keys": "alt+k" },
+        { "command": { "action": "movePane", "direction": "previous"}, "keys": "ctrl+alt+h" },

```

Comment:
I'd take these out of the default bindings for now. People really really dislike us camping keys like these. You could theoretically leave them here, but without `keys` set, so they still appear in the command palette.

Reply:
e.g.
```suggestion
        { "command": { "action": "movePane", "direction": "down" } },
        { "command": { "action": "movePane", "direction": "left" } },
        { "command": { "action": "movePane", "direction": "right" } },
        { "command": { "action": "movePane", "direction": "up" } },
        { "command": { "action": "movePane", "direction": "previous"} },
```